src/components/VideoPlayer/Player.js

```diff
@@ -302,7 +337,9 @@ class VideoPlayer extends Component {
     const { playing, progress, muted, subtitles, loading, fullscreen, isFullscreen } = this.state
 
     return (
-      <div {...merge(styles.wrapper, breakoutStyles[size])}>
+      <div {...merge(styles.wrapper, breakoutStyles[size])}
+        onClick={this.captureFocus.bind(this)}
+      >
         <video
           {...(isFullscreen ? styles.videoFullscreen : styles.video)}
```

Comment:
The focus triggers an outline on most browsers, e.g. a blue box shadow on Chrome:
<img width="674" alt="screen shot 2018-07-23 at 10 19 41" src="https://user-images.githubusercontent.com/23520051/43065803-3c0a1e06-8e63-11e8-8c12-572b41257b8f.png">

You can suppress that by applying
```
':focus': {
  outline: 'none'
}
```